moz_kinto_publisher/main.py

```diff
@@ -553,10 +559,10 @@ def publish_intermediates(*, args, rw_client):
         try:
```

Comment:
There's a line about 10 up that also uses `record["id"]` to delete an intermediate (thanks for not letting me comment on it, github 🙄) - does that need to change too?

Reply:
The one above is right, and it's the reason I initially made the type error here. The one above is an error case where we didn't create an `Intermediate` object, so we only have a dictionary. I cleaned it up a bit in the most recent commit.